src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarRegionSwitcher.test.tsx

```diff
@@ -16,20 +17,34 @@ jest.mock(
 );
 
 const props = {
+  display: DisplayType.FULL,
+  breakpointWidth: BreakpointWidth.TABLET,
   toggleRegionEditorActive: jest.fn(),
   toggleRegionFieldActive: jest.fn()
 };
 
 describe('BrowserNavBarRegionSwitcher', () => {
+  let wrapper: any;
+
+  beforeEach(() => {
+    wrapper = mount(<BrowserNavBarRegionSwitcher {...props} />);
+  });
+
   afterEach(() => {
     jest.resetAllMocks();
   });
 
-  it('renders correctly', () => {
-    const wrapper = mount(<BrowserNavBarRegionSwitcher {...props} />);
+  describe('rendering', () => {
+    it('renders only region field on smaller screens', () => {
```

Comment:
renders only region field on smaller screens -> renders region field only on smaller screens?

Reply:
This test should also say that region editor is not rendered. So will word it is as 'renders region field and not region editor on smaller screens'.

Reply:
:white_check_mark: